File: examples/python/token_profile.py
from e3d_client import E3DClient

TOKEN_ADDRESS = "0x6488861b401F427D13B6619C77C297366bCf6386"


def main() -> None:
    client = E3DClient()
    profile = client.get(f"/token/{TOKEN_ADDRESS}", query={"chain": "ETH"})
    print(profile)


if __name__ == "__main__":
    main()

File: examples/token-profile.mjs
import { E3D } from '../dist/index.js';

const tokenAddress = process.env.E3D_TOKEN_ADDRESS || '0x6488861b401F427D13B6619C77C297366bCf6386';
const e3d = new E3D({ baseUrl: process.env.E3D_BASE_URL || 'https://e3d.ai/api' });

const profile = await e3d.tokens.getTokenProfile(tokenAddress, { chain: 'ETH' });
console.log(JSON.stringify(profile, null, 2));
